.github/scripts/pr_body_check.py

```
@@ -0,0 +1,87 @@
+import os
+import re
+import sys
```

Ruff (0.12.2)

`sys` imported but unused

Remove unused import: `sys`

(F401)